### DISCUSSION TITLE
Fuseki-geosparql?

### DISCUSSION THREAD

**Discussion by tomkxy** [afs/rdf-delta#82] [General]
I would require to se Fuseki Geosparql. I was able to build it by replacing the dependencies to jena-fuseki by jena-fuseki-geosparql. Is there any reason why this could cause problems?

**Comment by afs** [comment]
Yes, it should work.

Just make sure the stack of datasets is changes > geosparql > base data so that the replaying of the patches across the servers goes through the GeoSPARQL dataset so all copies see the triple changes.

**Reply by tomkxy** [reply]
Although I understand it on a conceptual level I am not sure what it mean in terms of configuration or setup. I replaced in the poms the jena-fuseki lib by jena-fuseki-geosparql. Is there anything else to be done?

**Reply by afs** [reply]
I though the jena-geosparql module had an assembler and could be used for composite configurations (and not use jena-fuseki-geosparql). However, it doesn't. Sorry - I should have checked.

It would take java code to configure it, maybe the code in GeosparqlServer.

What sort of setup are you trying to build? If the updates are in SPARQL Update and come from one single place, there is a crude way to achieve a similar system - send the update to each replica rather than use Delta.. Not perfect but can give a 24/7 available service for other read/query applications.
